images/mongodb/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
+# mongodb
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/mongodb` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/mongodb/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+The MongoDB Database image
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/mongodb:latest
+```
+<!--getting:end-->
+
+<!--body:start--><!--body:end-->
```

Comment:
Since this is a new image it would be handy to have some documentation about how to use it. The Mongo docs are pretty extensive, so it doesn't need to be anything too in depth.

Very basic docker run, docker exec, run `mongo` and query or insert a document kind of steps would be enough.